docs/source/dataframe-joins.rst

```diff
@@ -58,8 +63,29 @@ maintains that index, like Parquet.
 
 .. code-block:: python
 
-    left = left.set_index('id').persist()
+    from dask.datasets import timeseries
+    from dask.dataframe import read_parquet
```

Comment:
A more standard import would be:

```suggestion
    import dask
    import dask.dataframe as dd
```

Then just use `dask.datasets.timeseries` and `dd.read_parquet`